scripts/report_missing_files.py

```diff
@@ -1,43 +1,75 @@
 import botocore
+import json
 import requests
+from multiprocessing import Pool, Value, Lock
 import sys
 
 from managers import DBManager, S3Manager
 from model import Part, Record
 
 
 '''
-Usage:  python main.py --script ReportFileAccessibility -e <env> options source=<source>
+Usage:  python main.py --script ReportMissingFiles -e <env> options source=<source>
 '''
-def main(*args):
-    db_manager = DBManager()
-    db_manager.createSession()
 
+record_count = Value('i', 0)
+missing_file_count = Value('i', 0)
+lock = Lock()
+
+
+def is_record_missing_files(record_parts: tuple) -> bool:
     s3_manager = S3Manager()
     s3_manager.createS3Client()
 
+    parts, *_ = record_parts
+
+    for part in parts:
+        index, url, source, file_type, flags = part.split('|')
```

Comment:
Could be added as a constructor on `Part` if we are constructing from string in multiple places

Reply:
Good callout - we definitely want to move away from this pipe delimiter in the data model. We will get there lol.